### PR TITLE
FIX: CU_SAFE_CALL used with cublasXXX func

### PR DESCRIPTION
**Pull request by selaselah**
all CuBLAS func return a cublasStatus_t, which is not same as cudaError_t and the value mean is not compatible

### CODE REVIEW

File: src/cudamatrix/cu-common.cc [hunk 1]
```diff
@@ -67,6 +67,25 @@ void GetBlockSizesForSimpleMatrixOperation(int32 num_rows,
                "Matrix has too many rows to process");
   dimGrid->z = 1;
 }
+
+const char* cublasGetStatusString(cublasStatus_t status)
```
Comment (danpovey) [inline src/cudamatrix/cu-common.cc]: thanks for the PR.
So there wasn't an official version of this function, that we could have used?

Comment (danpovey) [reply src/cudamatrix/cu-common.cc]: I see that there wasn't an official function like this, but please use Google style w.r.t. open-braces: put them on the same line as the previous one, after a space, e.g.
`const char* cublasGetStatusString(cublasStatus_t status) {`

And please put a comment before the declaration of this function, saying:
`// This is analogous to the CUDA function cudaGetErrorString().`

**Comment by selaselah** [comment]
the CI failed randomly. I have reopened this request several times but all are failed. Here is some error log:

    ln: failed to create symbolic link ‘/home/travis/build/kaldi-asr/kaldi/src/lib/libkaldi-matrix.so’: File exists

File: src/cudamatrix/cu-common.cc (hunk 1 above)
Comment (selaselah) [reply src/cudamatrix/cu-common.cc]: Reformat done.
Howerver, the CI raise some reasonless error :(

**Comment by danpovey** [comment]
Thanks.  Maybe @jtrmal or @kkm can look into the CI errors-- probably an
issue with the latest scripts there.

**Comment by jtrmal** [comment]
The compilation went fine, only the scripts weren't  able to finish storing
the ccache on time -- not sure what to do with that.
y.

**Comment by danpovey** [comment]
We have some CuSparse library calls in the 'rnnlm' branch and I think the same issue exists there.  Do you have time to address that via another PR?

**Comment by selaselah** [comment]
I check the CuSparse's doc,. The status value also conflict with cudaError_t. I'll take a PR tomorrow.

**Pull request merged**